src/components/core/DpEditor/DpEditor.vue

```diff
@@ -441,97 +393,22 @@ export default {
       default: ''
     },
 
-    /**
-     * If true, the button to add images will be shown and the initial text will be scanned for img placeholders which will be then replaced by actual images.
```

Comment:
Great to see those deprecations be removed! However the documentation should be kept somewhere. 

While typescript would make it easier to document nested object properties, i'd suggest to just move the docblocks down to where `this.toolbar` is assigned its default values, as seen in https://stackoverflow.com/a/19336366/6234391

Reply:
Done in [5474707](https://github.com/demos-europe/demosplan-ui/pull/104/commits/5474707014b90424f008d9f35613820829fb6ea1)